h/static/scripts/annotator/main.js

```diff
@@ -29,7 +29,8 @@ var docs = 'https://h.readthedocs.org/en/latest/hacking/customized-embedding.htm
 var options = {
   app: jQuery('link[type="application/annotator+html"]').attr('href'),
   BucketBar: {container: '.annotator-frame', scrollables: ['body']},
-  Toolbar: {container: '.annotator-frame'}
+  Toolbar: {container: '.annotator-frame'},
+  clickToClose: true
```

Comment:
Can you move this into the `options` object on line 37 of `guest.coffee` instead? We should maybe move all the defaults into there, but for the moment let's just not add more noise here.